managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -2333,6 +2338,9 @@ public void closeFailed(ManagedLedgerException exception, Object ctx) {
      */
     void persistPositionWhenClosing(PositionImpl position, Map<String, Long> properties,
             final AsyncCallbacks.CloseCallback callback, final Object ctx) {
+        log.info("Persistence before cursor close is in progress. position={} "
```

Comment:
I'm not sure that we need to add this log, it seems that there are logs when this operation was completed successfully or not.

Reply:
Move this log to `ManagedCusrorImpl#buildIndividualDeletedMessageRanges `.